compiler-rt/lib/rtsan/rtsan_interceptors.cpp

```diff
@@ -0,0 +1,412 @@
+//===--- rtsan_interceptors.cpp - Realtime Sanitizer ------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+//===----------------------------------------------------------------------===//
+
+#include "rtsan/rtsan_interceptors.h"
+
+#include "sanitizer_common/sanitizer_platform.h"
+#include "sanitizer_common/sanitizer_platform_interceptors.h"
+
+#include "interception/interception.h"
+#include "rtsan/rtsan_context.h"
+
+#if !SANITIZER_LINUX && !SANITIZER_APPLE
```

Comment:
If other OSes are not supported, the NetBSD/FreeBSD/etc references in `lit.cfg.py` should be removed.